borg/helpers.py

```diff
@@ -259,7 +259,7 @@ def update_excludes(args):
 
 def adjust_patterns(paths, excludes):
     if paths:
-        return (excludes or []) + [IncludePattern(path) for path in paths] + [ExcludePattern('*')]
+        return (excludes or []) + [PathPrefixPattern(path) for path in paths] + [FnmatchPattern('*')]
```

Comment:
due to the renaming, the code is now harder to understand.
one has to know which matching mechanism are used for incuding / for excluding.
also: why can't we have any mechanism as well for including as for excluding?